mappings/net/minecraft/inventory/Inventory.mapping

```diff
@@ -1,28 +1,28 @@
 CLASS agw net/minecraft/inventory/Inventory
-	METHOD K_ getInvMaxStackAmount ()I
-	METHOD N_ getInvSize ()I
-	METHOD a getInvStack (I)Lbcb;
+	METHOD K_ getSlotCapacity ()I
+	METHOD N_ getInventorySize ()I
+	METHOD a getStack (I)Lbcb;
 		ARG 1 slot
-	METHOD a takeInvStack (II)Lbcb;
+	METHOD a removeStack (II)Lbcb;
 		ARG 1 slot
 		ARG 2 amount
-	METHOD a setInvStack (ILbcb;)V
+	METHOD a setStack (ILbcb;)V
 		ARG 1 slot
 		ARG 2 stack
-	METHOD a canPlayerUseInv (Lavy;)Z
+	METHOD a canUseContainer (Lavy;)Z
 		ARG 1 player
-	METHOD a getInvAmountOf (Lbbw;)I
+	METHOD a getTotalCount (Lbbw;)I
 		ARG 1 item
-	METHOD a containsAnyInInv (Ljava/util/Set;)Z
+	METHOD a anyMatch (Ljava/util/Set;)Z
 		ARG 1 items
-	METHOD a_ onInvOpen (Lavy;)V
+	METHOD a_ onContainerOpened (Lavy;)V
```

Comment:
It should be `onOpened`, or `onInventoryOpened`. Some inventories don't have containers, like the player inventory.